layout-per-window.py

```diff
@@ -1,49 +1,76 @@
 #!/usr/bin/env python
 
 # This script keeps track of the active layout for each window.
+# Optional argument defines numeric layout index for new windows (counted from 0)
 #
 # This script requires i3ipc-python package (install it from a system package
 # manager or pip).
 
+import sys
```

Comment:
Need a fix

Reply:
Does this mean putting sys import after i3ipc-python? I thought third-party imports should come after stdlib imports.

Reply:
Just a newline will do, it's a lint error